src/applications/disability-benefits/all-claims/pages/medicalCare.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+const medicalDescription = (
+  <div>
+    <p>
+      Did you spend time in a hospital or under a doctor's care for your
+      service-connected disabilties in the past 12 months?
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': 'Medical Care',
+
```

Comment:
Use `import { unemployabilityTitle } from '../content/unemployabilityFormIntro';` for the main title, See https://github.com/department-of-veterans-affairs/vets-website/blob/bah-8940-doctor-interview/src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js

Reply:
ya. need to change since talking about it in round table. will do!

Reply:
<img width="601" alt="screen shot 2019-01-08 at 10 31 36 am" src="https://user-images.githubusercontent.com/41440372/50840477-9f17b600-1330-11e9-8d39-9bb3da424b9f.png">